mouse/picture.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import print_function
+
+__author__ = "Bailey Nozomu Hwa <hwab@janelia.hhmi.org>"
+__date__ = "$Aug 05, 2015, 3:36 PM$"
+
+import argparse
+import io
+import picamera
+import datetime
+import os
+
+
+def main(*argv):
+    """
+        Simple main function that takes a photo of the background when run.
+
+        Args:
+            argv(str):                      Arguments are stored as a list
+
+        Notes:
+            these are the parameters of the arguments.
+
+            folder(str):                    File directory
+
+
+    """
+
+    # Directory is stored here
+    parser = argparse.ArgumentParser(
+        description="Directory to save the picture into"
```

Comment:
Should explain what the program does. Takes a picture and saves it to a file. Also, add a period at the end.